_data/doc_version_index.yml

```diff
@@ -1,6 +1,7 @@
 # Auto-generated by ./scripts/build_doc_version_index.sh
```

Comment:
Do we need to have this file as part of this commit?

Reply:
Nope